app/controllers/wpcc/engine_controller.rb

```diff
@@ -1,9 +1,15 @@
 module Wpcc
   class EngineController < Wpcc.parent_controller.constantize
+    WPCC_SESSION_EXPIRY_LIMIT = (30 * 60)
```

Comment:
yes and I considered that, but when I was thinking about env vars and yml files, I thought that 30.minutes wouldn't work in those contexts. If you think 30.minutes is better and it is certainly more readable, happy to change *(after lunch)*.